system/security/shared/caching.cpp

```diff
@@ -504,7 +505,7 @@ bool CPermissionsCache::addManagedFileScopes(IArrayOf<ISecResource>& scopes)
 
 inline void CPermissionsCache::removeManagedFileScopes(IArrayOf<ISecResource>& scopes)
 {
-    WriteLockBlock writeLock(m_scopesRWLock);
+    //WriteLockBlock writeLock(m_scopesRWLock);
```

Comment:
worth changing this to a comment, stating that this method must be called while m_scopesRWLock lock held.

NB: looks like this method is never used

Reply:
in general we are better off removing code that isn't used, and hasn't been for years, we can always reintroduce it if there's a need.